mscore/musescore.cpp

```diff
@@ -6018,6 +6018,32 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   cv->setMag(MagIdx::MAG_100, 1.0);
             setMag(1.0);
             }
+      else if (cmd == "zoom-page-width") {
+            if (cv) {
+                  // Store current zoom-level statically here for later retrieval
+                  // instead of implementing it as a member variable since there's
+                  // (currently) no other use for it.
+                  MagIdx current_idx = MagIdx(mag->currentIndex());
+                  static MagIdx previous_idx = current_idx;
```

Comment:
`previous_idx` (or rather `previousIdx`, as per comment above) should probably logically belong to the score view rather than to `MuseScore::cmd()` function so it should probably be made a member of `ScoreView` class. The current implementation would probably cause inconsistent behavior if this shortcut is invoked in several score tabs.

Reply:
After testing three tabs with this shortcut, there doesn't seem to be an issue with variations of this shortcut. But I slightly understand why you say the variable should be stored elsewhere. Since nothing else is using this information, figured it would be okay, but i'll make adjustments + the camel case if this is what's decided.

Reply:
After setting up the code to be related to handbook + moving variable, I have found there to be one slight 'bug' here, but it's a niche use case and would be more complex to alleviate it: there can be multiple scores open with toggling, but if there is more than one custom zoom level rather than a level from the list, then if the order of toggling overlaps, i.e., if two scores are in page-width view but on the 'other side of the toggle' have two completely different custom zoom levels, toggling in one of them will end up changing the other's custom zoom level when toggling back to it in the other score. 

I think it's still a valid implementation still since this is probably rare activity, and as the zoom table is currently implemented, there is only one slot for a "FREE_MAG" data and only one double variable for that zoom level. For now I think it will work fine unless others end up complaining about it in the forums, and then I'd be willing to spend more time implementing another table of some sort that stored MagIdx's with corresponding scores to allow for always retained custom zoom levels, but I think for now that's overkill, unless i'm overlooking something. And again, if the user sticks with the drop-down zoom levels, this won't happen at all and still allows for *one* custom "FREE_MAG" zoom level with zero problems during toggling